### PR TITLE
Version 10 compatibility

### PR DESCRIPTION
**Pull request by Zetat**

### CODE REVIEW

**Comment by tomolimo** [comment]
Hello @Zetat 
Thank you for this PR,
I've two questions:
1. Are you sure that it will provide compatibility with GLPI 10?
1. For the min max, for min the test is "greater or equal", but for max it is "less than", so for me if you want to run this plugin with GLPI 10, then max should be 11, and not 10.

Could you check these?
Thank you,
Regards,
Tomolimo

File: setup.php [hunk 1]
```diff
@@ -44,7 +44,7 @@ function plugin_version_mailanalyzer() {
       'requirements' => [
          'glpi' => [
             'min' => '9.5.3',
```
Comment (tomolimo) [inline setup.php:46]: this must be 10.0

File: setup.php [hunk 2]
```diff
@@ -44,7 +44,7 @@ function plugin_version_mailanalyzer() {
       'requirements' => [
          'glpi' => [
             'min' => '9.5.3',
-            'max' => '9.6'
+            'max' => '10'
```
Comment (tomolimo) [inline setup.php:47]: this must be 10.1

**Comment by iemilio** [comment]
> 2\. be 11, and not 10.



> Hello @Zetat Thank you for this PR, I've two questions:
> 
> 1. Are you sure that it will provide compatibility with GLPI 10?
> 2. For the min max, for min the test is "greater or equal", but for max it is "less than", so for me if you want to run this plugin with GLPI 10, then max should be 11, and not 10.
> 
> Could you check these? Thank you, Regards, Tomolimo

Hey, we changed the max to 11 and plugin its enabled, thanks a lot

**Pull request closed without merging**